src/gui/guiFormSpecMenu.cpp

```diff
@@ -1936,6 +2124,11 @@ void GUIFormSpecMenu::parseElement(parserData* data, const std::string &element)
 		return;
 	}
 
+	if (type == "real_coordinates") {
+		data->real_coordinates = is_yes(description);
```

Comment:
I think, it would be better having here a `parseRealCoordinates` function as every other element type does the same.

Reply:
It was a function before, but SmallJoker suggested that it be inlined since it's only a single line.  I could make it a macro, but I don't see the point, really.

Reply:
(Or you could define the function as `inline`.)
But you are right, this is probably ok like it is now.
https://github.com/minetest/minetest/pull/8524#discussion_r283117155